src/internal_events/exec.rs

```diff
@@ -43,6 +43,10 @@ impl InternalEvent for ExecEventsReceived<'_> {
     }
 }
 
+// only used by exec
+// should be moved to src/internal_events/prelude.rs if used somewhere else
+pub const COMMAND_FAILED: &str = "command_failed";
```

Comment:
Could we just proactively move it there? I think that'll make it more discoverable if anther source does find a use for it. Otherwise I could see someone introducing a similar, but different event type because they didn't know this one existed.

This way we start to build up a catalog of error types for events.